### PR TITLE
Rebrand Thycotic to Delinea

### PR DESCRIPTION
**Pull request by gaurava-delinea**
Required items, please complete
   
   Change(s):
   - See guidance below

   Reason for Change(s):
   - See guidance below

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   - See guidance below

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

# Guidance <- remove section before submitting
-----------------------------------------------------------------------------------------------------------
## Before submitting this PR please ensure that you have read the following sections and filled out the changes, reason for change and testing complete sections:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

   Version updated:
   - Yes
   - Detections/Analytic Rule templates are required to have the version updated

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
   - Yes/No/Need Help

_Note: If updating a detection, you must update the version field._

> Before the submission has been made, please look at running the KQL and Yaml Validation Checks locally.
> https://github.com/Azure/Azure-Sentinel#run-kql-validation-locally

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes/No/Need Help
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

**Comment by tylerezimmerman** [comment]
@v-sabiraj I have signed the CLA in a different PR for Microsoft Power automate on behalf of Delinea. @gaurava-delinea works on my team. How can we proceed with that license check? Can it be assigned to me?

**Comment by v-sabiraj** [comment]
Hey @gaurava-delinea, can you please sign it once again from here? as check is blocked because of that, thanks.

**Comment by tylerezimmerman** [comment]
@v-sabiraj when can we expect this PR to be accepted and merged?

**Comment by v-sabiraj** [comment]
@gaurava-delinea, there is error in workbook, please check and update, thanks.
image

File: Solutions/Delinea Secret Server/Data Connectors/DelineaSecretServer_CEF.json [hunk 1]
```diff
@@ -1,42 +1,42 @@
...
+    "id": "DelineaSecretServer_CEF",
+    "title": "Delinea Secret Server",
+    "publisher": "Delinea, Inc",
+    "descriptionMarkdown": "Common Event Format (CEF) from Delinea Secret Server ",
     "graphQueries": [
         {
             "metricName": "Total data received",
-            "legend": "ThycoticSecretServer",
-            "baseQuery": "\nCommonSecurityLog\n| where DeviceVendor == \"Thycotic Software\"\n| where DeviceProduct == \"Secret Server\"\n"
+            "legend": "CommonSecurityLog (Delinea Secret Server)",
```
Comment (v-sabiraj) [inline Solutions/Delinea Secret Server/Data Connectors/DelineaSecretServer_CEF.json:9]: @gaurava-delinea, legend should be "DATATYPE_NAME", there should not be spaces in datatype name, please check and update, thanks.

File: Solutions/Delinea Secret Server/Data Connectors/DelineaSecretServer_CEF.json [hunk 2]
```diff
@@ -1,42 +1,42 @@
...
             "description" : "Get records where view secret",
-            "query" :"CommonSecurityLog\n| where DeviceVendor == \"Thycotic Software\"\n| where DeviceProduct == \"Secret Server\"\n| where Activity contains \"SECRET - VIEW\""
+            "query" :"CommonSecurityLog\n| where DeviceVendor == \"Delinea Software\"\n| where DeviceProduct == \"Secret Server\"\n| where Activity contains \"SECRET - VIEW\""
         }
     ],
     "dataTypes": [
         {
-            "name": "CommonSecurityLog (ThycoticSecretServer)",
-            "lastDataReceivedQuery": "\nCommonSecurityLog\n| where DeviceVendor == \"Thycotic Software\"\n| where DeviceProduct == \"Secret Server\"\n\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+            "name": "CommonSecurityLog (Delinea Secret Server)",
```
Comment (v-sabiraj) [inline Solutions/Delinea Secret Server/Data Connectors/DelineaSecretServer_CEF.json:25]: @gaurava-delinea, there should not be spaces between datatype name, thanks.

**Review by v-sabiraj — CHANGES_REQUESTED** [review]

**Comment by v-sabiraj** [comment]
Hey @gaurava-delinea, please update Sample data files, [ThycoticSourceData.csv](https://github.com/Azure/Azure-Sentinel/blob/master/Sample%20Data/CEF/ThycoticSourceData.csv) and [ThycoticSourceData.json](https://github.com/Azure/Azure-Sentinel/blob/master/Sample%20Data/CEF/ThycoticSourceData.json), thanks.

File: Solutions/Delinea Secret Server/Data Connectors/DelineaSecretServer_CEF.json (hunk 1 above)
Comment (gaurava-delinea) [reply Solutions/Delinea Secret Server/Data Connectors/DelineaSecretServer_CEF.json]: chnaged the legend from CommonSecurityLog (Delinea Secret Server) to CommonSecurityLog(DelineaSecretServer)

**Comment by tylerezimmerman** [comment]
@v-sabiraj the changes were submitted by @gaurava-delinea in the most recent commit. Please evaluate.

**Comment by v-sabiraj** [comment]
> @gaurava-delinea, there is error in workbook, please check and update, thanks. image

@gaurava-delinea / @tylerezimmerman, still the workbook is not visible after deployment, can you check on this

**Comment by v-sabiraj** [comment]
> > @gaurava-delinea, there is error in workbook, please check and update, thanks. image
> 
> @gaurava-delinea / @tylerezimmerman, still the workbook is not visible after deployment, can you check on this

@gaurava-delinea, the possible reason for this should be the schema, please check and update "version": "Notebook/1.1", thanks.

**Comment by tylerezimmerman** [comment]
@v-sabiraj the change was updated by @gaurava-delinea

File: Solutions/Delinea Secret Server/Workbooks/DelineaWorkbook.json [hunk 3]
```diff
@@ -315,8 +315,8 @@
     }
   ],
   "fallbackResourceIds": [
-    ""
+    "/subscriptions/80e05ab0-5906-45a8-98ff-c9520c4b418b/resourcegroups/thycotic/providers/microsoft.operationalinsights/workspaces/thycotic02"
```
Comment (v-sabiraj) [inline Solutions/Delinea Secret Server/Workbooks/DelineaWorkbook.json:318]: @gaurava-delinea, @tylerezimmerman, workbook validations are failing as the workbook contains info of a resource. A workbook template must not contain any references to resources. Please check and update, thanks.

**Comment by v-sabiraj** [comment]
@gaurava-delinea, the workbook and data connector queries are using Delinea software but in sample data it is still thycotic software, please update sample data file accordingly, thanks. 
image

**Comment by v-sabiraj** [comment]
@gaurava-delinea, can you please add updated workbook preview images, thanks.

**Comment by gaurava-delinea** [comment]
> @gaurava-delinea, the workbook and data connector queries are using Delinea software but in sample data it is still thycotic software, please update sample data file accordingly, thanks. image

@v-sabiraj @tylerezimmerman Done the additional commit. Also find some more places where Thycotic references being used in many places. Kindly review it.

**Comment by tylerezimmerman** [comment]
@v-sabiraj please review the most recent commit. We are unable to generate any rebranded images until this PR is approved and we can view it on the preview instance. Then we can submit another PR with the updated changes.

File: Solutions/Delinea Secret Server/Workbooks/DelineaWorkbook.json (hunk 3 above)
Comment (v-sabiraj) [reply Solutions/Delinea Secret Server/Workbooks/DelineaWorkbook.json]: > @gaurava-delinea, @tylerezimmerman, workbook validations are failing as the workbook contains info of a resource. A workbook template must not contain any references to resources. Please check and update, thanks.

Hey @gaurava-delinea , @tylerezimmerman , can you please fix this, all validations should pass for the PR to be merged.

**Comment by v-sabiraj** [comment]
@gaurava-delinea, @tylerezimmerman, workbook validations are failing as the workbook contains info of a resource. A workbook template must not contain any references to resources. Please check and update, thanks.

**Comment by gaurava-delinea** [comment]
> @gaurava-delinea, @tylerezimmerman, workbook validations are failing as the workbook contains info of a resource. A workbook template must not contain any references to resources. Please check and update, thanks.

@v-sabiraj This comment is not clear to me. could you please highlight the line that is not needed.

**Comment by v-sabiraj** [comment]
Hi @gaurava-delinea, the value for fallbackResourceIds in workbook refers to some resources, initially it was just "", so can you check on that? 
image

**Comment by gaurava-delinea** [comment]
> Hi @gaurava-delinea, the value for fallbackResourceIds in workbook refers to some resources, initially it was just "", so can you check on that? image

@v-sabiraj @tylerezimmerman 
Pushed the latest workbook file. comment Resolved. Please approve the PR

**Comment by tylerezimmerman** [comment]
@v-sabiraj any update here?

**Comment by v-sabiraj** [comment]
Hey @tylerezimmerman/@gaurava-delinea, the checks are failing, not sure why the validations are failing for the files that are not added in the PR, I'll check on this, thanks.
image

**Comment by v-spadarthi** [comment]
@gaurava-delinea : Please replace "Azure Sentinel" to Microsoft Sentinel" where ever it's applicable.

**Comment by gaurava-delinea** [comment]
> @gaurava-delinea : Please replace "Azure Sentinel" to Microsoft Sentinel" where ever it's applicable.

@v-sabiraj @tylerezimmerman resolved the comment

**Comment by tylerezimmerman** [comment]
@v-sabiraj please review the recently uploaded changes.

**Review by v-sabiraj — APPROVED** [review]
Looks good

**Review by NikTripathi — APPROVED** [review]

**Pull request merged**